components/org.wso2.carbon.identity.api.server.webhook.management/org.wso2.carbon.identity.api.server.webhook.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/webhook/management/v1/model/WebhookSummary.java

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.webhook.management.v1.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.xml.bind.annotation.*;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public class WebhookSummary  {
+  
+    private String id;
+    private String createdAt;
+    private String updatedAt;
+    private String endpoint;
+    private String name;
+    private List<String> eventsSubscribed = null;
```

Comment:
Refer my comments above and we don't need to have eventsSubscribed here

Reply:
fixed with https://github.com/wso2/identity-api-server/pull/902/commits/0386f0e0fc36edb1fa5b62bd5ea86eb2765b3487